umake/frameworks/go.py

```diff
@@ -42,13 +42,11 @@ def __init__(self):
 
 class GoLang(umake.frameworks.baseinstaller.BaseInstaller):
 
-    def __init__(self, category):
+    def __init__(self, **kwargs):
         super().__init__(name="Go Lang", description=_("Google compiler (default)"), is_category_default=True,
-                         category=category, only_on_archs=['i386', 'amd64'],
-                         download_page="https://golang.org/dl/",
-                         checksum_type=ChecksumType.sha1,
-                         dir_to_decompress_in_tarball="go",
-                         required_files_path=[os.path.join("bin", "go")])
+                         only_on_archs=['i386', 'amd64'], download_page="https://golang.org/dl/",
```

Comment:
Please don't add additional diff that's not in context :)
I keep some elements alone in their lines when there are multiple of them for easier readiness (that comments is for all frameworks). Here download_page and checkstum_type for instance.

Please only remove category=category and add **kwargs at the end with minimal formatting (that remark is for all frameworks)